src/js/utils/ui.js

```diff
@@ -109,7 +109,7 @@ function initInteractionListeners(ui) {
 
         removeHandlers(ui, WINDOW_GROUP);
         if (type === 'pointerdown' && e.isPrimary) {
-            if (!passive) {
```

Comment:
Does this work everywhere?

I was thinking we defer pointer capture https://github.com/jwplayer/jwplayer/compare/wip/ui-pointer-capture-when-dragging?expand=1